pom.xml

```diff
@@ -23,7 +23,7 @@
         <dependency>
             <groupId>org.jitsi</groupId>
             <artifactId>rtp</artifactId>
-            <version>1.0-20190723.115603-38</version>
+            <version>1.0-0-gfde2add</version>
```

Comment:
@bgrozev this is not the scheme we laid out as i understood it.  shouldn't this just be `1.0` or `1.1`?

Reply:
sorry, misread this.  ignore me.